app/views/apps/parliament_vote_app/application_forms/delivery_address.html.erb

```diff
@@ -4,7 +4,8 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <%= form_for @application_form, :builder => AppFormBuilder, url: request.original_url do |f| %>
-      <%= render 'apps/parliament_vote_app/application_forms/back', f: f %>
+      <%= render 'apps/parliament_vote_app/application_forms/back', form: f %>
+      <%= render 'components/error_summary', form: @application_form %>
```

Comment:
mozno ako helper `f.error_summary` by to tiez nebolo na zahodenie.

Reply:
Super napad!